src/compas/topology/orientation.py

```diff
@@ -117,10 +121,12 @@ def face_adjacency(points, faces):
     purely geometrical, but uses a spatial indexing tree to speed up the search.
 
     """
-    f = len(faces)
-
-    if f > 100:
-        return _face_adjacency(points, faces)
+    if nmax is not None:
+        if radius is not None:
+            return _face_adjacency(points, faces, nmax=nmax, radius=radius)
+        return _face_adjacency(points, faces, nmax=nmax)
+    if radius is not None:
+        return _face_adjacency(points, faces, radius=radius)
```

Comment:
perhaps it would make sense to refactor `_face_adjacency` such that you can write here
```python
if nmax or radius:
    return _face_adjacency(points, faces, nmax=nmax, radius=radius)
```